governance.md

```diff
@@ -24,17 +28,53 @@ Every code owner is expected to:
 * Approve only those PRs in which the contributor made the requested improvements
 * Check if the related CI tests have completed successfully before approving the PR
 * Make sure that the PR approval flow runs smoothly
-* Prioritize issues and manage their processing
 * Proactively fix bugs
 * Perform maintenance tasks for their projects
-
-### Add or remove a code owner
+* Issue that is assigned to current release is treated with priority by maintainer that is an assignee
+* Monitor Slack
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community. Follow [labels workflow](/governance/#issues-workflow-issues-workflow) document
+* In general continue to be willing to spend at least 50% of one's time working on Kyma (~2.5 business days per week).
+
+### Process for becoming a maintainer aka codeowner
+
+* Express interest to the existing maintainers that you or your organization are interested in becoming a
+  maintainer. The difference is that in case you are individual or corporate contributor/maintainer then different contributor license agreement applies.
+* Express what are the areas of Kyma that you want to engage with. Kyma, because of its goal, is complex and it is impossible to become a maintainer in every area. Make sure you get support from proper group of maintainers.
+* Becoming a maintainer generally means that you are going to be spending substantial
```

Comment:
that is a detailed metric, that could be measured long term.

is it enough to specify 2.5day a week? or you really want to go deeper into metrics you mentioned. I honestly prefer to go small and extend in case we see problems that maintainers join, number grows but none is active

Reply:
I do not have any problem with 
>  "go small and extend in case we see problems" 

but right now I see a problem with "work 2.5day a week" metrics because I really do not know how you will measure/check that. It is not described anywhere in this document, so for me is only a statement in docs but in real life, no one will check that because we will not have any tools to do it.

Reply:
how about 

```
Becoming a maintainer generally means that you are going to be spending substantial time (>50%) on Kyma for the foreseeable future. Among other responsibilities this is:
- creating and merging pull-request
- reviewing pull-request
- creating or commenting on issues
- helping on Slack and other community communication channels
```

Reply:
@derberg yep, pico bello 👍